torch/csrc/stable/tensor-inl.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
See my above comment. Use either CamelCase or snake_case

Reply:
okay, ill switch to snake case for these